infra/hoogle_server.tf

```diff
@@ -197,7 +214,7 @@ resource "google_compute_backend_service" "hoogle-http" {
   port_name     = "http"
 
   backend {
-    group = google_compute_instance_group_manager.hoogle.instance_group
+    group = google_compute_instance_group_manager.hoogle[0].instance_group
```

Comment:
How does the index work here? in `h_clusters`, index 0 is the deactivated blue deployment whereas I was expecting to see this point to green. What am I missing?

Reply:
It's me who missed this: it should be a dynamic block like the https one. Updated.